pkg/model/components/etcdmanager/model.go

```diff
@@ -449,6 +449,24 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster *kops.EtcdClusterSpec) (*v1.Po
 				},
 			},
 		})
+
+		if fi.BoolValue(b.Cluster.Spec.MountCertificates) {
+			container.VolumeMounts = append(container.VolumeMounts, v1.VolumeMount{
+				Name:      "ca-certs",
```

Comment:
There might be more of these directories in future (no two distros can agree it seems!), so maybe name it "etc-ssl-certs"?